website/install.md

```diff
@@ -21,14 +28,25 @@ Git Town runs anywhere Git and [Bash](https://www.gnu.org/software/bash/bash.htm
     </td>
     <td>
       <ul>
+        <li>On Arch Linux, install <code><a href="https://aur.archlinux.org/packages/git-town/">git-town</a></code> from the AUR</li>
```

Comment:
Can you possibly add the exact command to run here? Merely telling people to "install git-town from the AUR" is a bit vague.